admin-dev/themes/new-theme/js/pages/product/edit/combinations-manager.ts

```diff
@@ -285,56 +351,87 @@ export default class CombinationsManager {
     this.eventEmitter.on(CombinationEvents.bulkUpdateFinished, () => this.refreshPage());
   }
 
-  private initSubmittableInputs() {
-    const combinationToken = this.getCombinationToken();
-    const {impactOnPriceKey, referenceKey, tokenKey, deltaQuantityKey} = CombinationsMap.combinationItemForm;
-
-    new SubmittableInput({
-      wrapperSelector: CombinationsMap.impactOnPriceInputWrapper,
-      submitCallback: input =>
-        this.combinationsService.updateListedCombination(
-          this.findCombinationId(input),
-          {
-            [impactOnPriceKey]: input.value,
-            [tokenKey]: combinationToken
-          }
-        )
+  private watchInputChange($input: JQuery, initialValue: string | number | string[]): void {
+    $input.data('initialValue', initialValue);
+    $input.toggleClass(ProductMap.combinations.list.modifiedFieldClass, $input.val() !== initialValue);
+    $input.on('change', () => {
+      $input.toggleClass(ProductMap.combinations.list.modifiedFieldClass, $input.val() !== $input.data('initialValue'));
     });
+  }
 
-    new SubmittableInput({
-        wrapperSelector: CombinationsMap.referenceInputWrapper,
-        submitCallback: input =>
-          this.combinationsService.updateListedCombination(
-            this.findCombinationId(input),
-            {
-              [referenceKey]: input.value,
-              [tokenKey]: combinationToken
-            }
-          )
-      },
-    );
+  private updateByPriceImpactTaxExcluded($priceImpactTaxExcluded: JQuery): void {
+    const $row = $priceImpactTaxExcluded.parents(ProductMap.combinations.list.combinationRow);
+    const $priceImpactTaxIncluded = $(ProductMap.combinations.list.priceImpactTaxIncluded, $row);
 
-    new SubmittableDeltaQuantityInput({
-      submittableWrapperSelector: CombinationsMap.tableRow.deltaQuantityWrapper,
-      submitCallback: input => this.combinationsService.updateListedCombination(
-        this.findCombinationId(input),
-        {
-          [deltaQuantityKey]: input.value,
-          [tokenKey]: combinationToken,
-        },
-      ),
-      containerSelector: `${CombinationsMap.combinationsListContainer} ${CombinationsMap.tableRow.deltaQuantityWrapper}`,
-    });
+    if (typeof $row === 'undefined' || typeof $priceImpactTaxIncluded === 'undefined') {
+      return;
+    }
+
+    // @ts-ignore
```

Comment:
Why do we need to ignore there? Because of BigNumber? We really need to get rid of this asap